test/Unit/Runners/PHPUnit/SuiteLoaderTest.php

```diff
@@ -364,4 +364,12 @@ public function testExecutableTestsForFunctionalModeUse(): void
         $testMethod = $tests[1];
         static::assertEquals($testMethod->getName(), 'testTwoA|testTwoBDependsOnA');
     }
+
+    public function testDataProviderWithParameter(): void
+    {
+        $path   = $this->fixture('passing-tests/DataProviderParameterTest.php');
+        $loader = new SuiteLoader();
+        $loader->load($path);
+        $loader->getTestMethods();
```

Comment:
You should add an assertion on what you expect from this calls

Reply:
@Slamdunk Yep, I'm trying to figure out your tests environment. Is this "testDataProviderWithParameter" needed anyway? As I can see, my test added in the passing-tests folder is being executed in functional tests, so maybe it should be sufficient to just leave test in passing-tests and remove my method from SuiteLoaderTest?

Reply:
Sounds reasonable: the code-coverage will tell you if your new lines are covered